model/__init__.py

```diff
@@ -336,6 +340,10 @@ def from_dict(d):
             cov_B_at_scan_points.reshape(flex.grid(xl.num_scan_points, 9, 9))
             xl.set_B_covariance_at_scan_points(cov_B_at_scan_points)
 
+        recalc_unit_cell = d.get("recalc_unit_cell")
+        if recalc_unit_cell is not None:
+            xl.set_recalc_unit_cell(recalc_unit_cell)
+
```

Comment:
Standard deviations need to be serialized?

Reply:
Yes, fix incoming